apps/meeting/src/components/MeetingControls/VideoInputTransformControl.tsx

```diff
@@ -51,28 +51,30 @@ const VideoInputTransformControl: React.FC<Props> = ({
     resetDeviceToIntrinsic();
   }, []);
 
-  // Reset the video input to intrinsic if current video input is a transform device because this component does not know if blur or replacement was selected.
-  // This depends on how the demo is set up.
-  // TODO: use a hook in the appState to track whether blur or replacement was selected before this component mounts.
-  async function resetDeviceToIntrinsic() {
+  // Reset the video input to intrinsic if current video input is a transform device because this component
+  // does not know if blur or replacement was selected. This depends on how the demo is set up.
+  // TODO: use a hook in the appState to track whether blur or replacement was selected before this component mounts,
```

Comment:
Why is this TODO there? Is it there to remove the `isVideoTransformDevice` call in the future?

Reply:
So we have a `VideoTransformDropdown` component in the device setup page. We can preview the camera with those background features. If we select a background feature there, the `selectedVideoInputDevice` in `MeetingManager` will be updated. However the operation of enabling background feature is not tracked anywhere. So after we enter the meeting page. Although we selected a background transform device (background feature is enabled), but for the other components, they don't know that the background feature is enabled. So this TODO is mostly to fix this issue.

> Is it there to remove the isVideoTransformDevice call in the future

Currently we check if the background feature is enabled by checking the type of `selectedVideoInputDevice`, the `isVideoTransformDevice` call here. This is not very ideal from a long term. If we finish this TODO item, I think we are able to get rid of it.